migration/src/spreadsheets.rs

```diff
@@ -171,13 +176,19 @@ impl SheetResult {
                     doc_string = row_values.next();
                 }
 
+                let mut intro_or_genre = Body;
+                if is_intro == true {
+                    intro_or_genre = Intro;
+                }
```

Comment:
Aim to avoid `mut` when possible (also doing `== true` with bools), you can do this instead:
```rs
let x = if is_intro { Intro } else { Body };
```

Reply:
cool, thanks-- I fixed this in the instance that you pointed out and rewrote a few other lines in parsing the collection index to reflect this.